documentation/CAMARA-OIDC-profile.md

```diff
@@ -0,0 +1,1022 @@
+
+      The request is missing a required parameter, includes an unsupported parameter value (other than grant type), repeats a parameter, includes multiple credentials, utilizes more than one mechanism for authenticating the client, or is otherwise malformed.
+
+    * **_invalid_client_**
+
+      Client authentication failed (e.g., unknown client, no client authentication included, or unsupported authentication method). The authorization server MAY return an HTTP 401 (Unauthorized) status code to indicate which HTTP authentication schemes are supported. If the client attempted to authenticate via the `Authorization` request header field, the authorization server MUST respond with an HTTP 401 (Unauthorized) status code and include the `WWW-Authenticate` response header field matching the authentication scheme used by the client.
+
+    * **_invalid_grant_**
+
+      The provided authorization grant authorization code is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client.
+
+    * **_unauthorized_client_**
+
+      The authenticated client is not authorized to use this authorization grant type.
+
+    * **_unsupported_grant_type_**
+
+      The authorization grant type is not supported by the authorization server.
+
+    * **_invalid_scope_**
+
+      The requested scope is invalid, unknown, malformed, or exceeds the scope granted by the resource owner.
+
+
+* **error_description**
+
+  OPTIONAL. Human-readable ASCII text providing additional information, used to assist the client developer in understanding the error that occurred. Values for the `error_description` parameter MUST NOT include characters outside the set %x20-21 / %x23-5B / %x5D-7E.
+
+
+The parameters are included in the entity-body of the HTTP response using the `application/json` media type.
+
+`error_description` parameter is not a valid field to take decisions on the developer side, as its value may vary depending on the server implementation, language, format or context. Therefore, the developer should base their logic only on the `error` parameter.
+
+The following is a non-normative example Token Error Response:
+
+```http
+HTTP/1.1 400 Bad Request
+Content-Type: application/json
+```
+```json
+{
+  "error": "unauthorized_client"
+}
+```
+
+## The Scope Parameter
+
+Oauth scope values determine the specific CAMARA services being requested by the Service Provider, subject to the SP being registered to use those services.
+
+The Authorization Request MUST contain the `scope` parameter, which is a space delimited, case-sensitive list of ASCII strings (scope values). Multiple scope values can be requested simultaneously, subject to the SP being registered to use those “scopes”.
+
+For OpenID requests, the scope values MUST include `openid`, to indicate that the request is an OpenID Connect request, followed by other values depending on the specific CAMARA services being requested.
+
```

Comment:
The rest of the information in this section is already in [CAMARA-API-access-and-user-consent](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md), which should be referenced and not just duplicated here. All that will happen is that the two descriptions will diverge if suggested changes are adopted in this document.

Reply:
Yes, I suggest removing contents form this document that is already in standards or in other other Camara documents.

Reply:
Removed duplicates from internal documentation. Right now the link to that information is broken, the new reference link will work when the new version (0.2.0) is created in the repository.